Shared/Samples/Find closest facility from point/FindClosestFacilityFromPointView.swift

```diff
@@ -0,0 +1,235 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct FindClosestFacilityFromPointView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// A Boolean value indicating whether a routing operation is in progress.
+    @State private var routing = false
```

Comment:
```suggestion
    @State private var isRouting = false
```
This Boolean does not read as an assertion. Feel free to use a different name than suggested.